src/lib/isISO4217.js

```diff
@@ -0,0 +1,38 @@
+import assertString from './util/assertString';
+
+// from https://en.wikipedia.org/wiki/ISO_4217
+const validISO4217CurrencyCodes = [
```

Comment:
perf: we can use a `Set` along with `.has` instead of `.indexOf`.

Reply:
Sure @fedeci. I used another validator as a sample that used `.indexOf`, but using `.has` simplifies to code as well. Thans for the suggestion.

Reply:
Probably other ones need to be updated too :)

Reply:
Nice! I'll work on that once we have this feature validated.